open-build-service/create_debian

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
Add comment and/or rename script to indicate it's not used by OBS directly, but is helpful for testing packaging locally

Reply:
This has been addressed in the latest commit #[3eaf213](https://github.com/notqmail/notqmail/pull/76/commits/3eaf2138ce40da0831bfc90852faa8b7c2a2c4ff)